polkadot.js

```diff
@@ -0,0 +1,727 @@
+// When we are done with all notes, this file should not have any `// comments` left, and we can turn it back to JSON.
+
+let json = [
+  {
+    "title": "Intro to Polkadot",
+    "index": 1,
+    "challengeSteps": [
+      {
+        "id": 1,
+        // Image uses old logo.
+        "imageUrl": "https://i.imgur.com/vF7XfK8.png",
+        "title": "Meet the next-generation blockchain",
+        // TODO: I think we should start moving away from just "blockchain" and talk more about Web3 applications and services.
+        // 3 pillars: security, scalability, interoperability. I think this should be the basis of what we teach.
+        // "Decentralized web" is still okay, but I think we want to start using "world computer"
+        "details": "Welcome to the world of Polkadot, a blockchain platform that enables communication between different blockchains - created by Gavin Wood, one of the co-founders of Ethereum. It’s able to connect with different blockchains to form an interoperable, decentralized web."
+      },
+      {
+        "id": 2,
+        "imageUrl": "https://i.imgur.com/9BQhqPY.png",
+        "title": "Building blocks of Polkadot",
+        // This feels to me more like describing the Polkadot SDK.
+        // When you imagine building on Polkadot, you imagine gaining all the flexibility without sacrificing anything when building a web3 application.
+        "details": "Think of Polkadot like building with Legos - you can pick and choose the pieces you want to create something unique! Imagine building a hyper-optimized blockchain, called a parachain, with specific characteristics to fit your exact need! All this is possible with Polkadot!"
```

Comment:
This feels to me more like describing the Polkadot SDK.
When you imagine building on Polkadot, you imagine gaining all the flexibility without sacrificing anything when building a web3 application.